### DISCUSSION TITLE
Any way to detect whether you're using the web-based editor or VS Code?

### DISCUSSION THREAD

**Discussion by danfarinoeyecue** [community/community#12526] [Codespaces]
Hi, is there any command or API available that can tell me whether I am running inside the web-based editor or inside VS Code?

When running in the web-based editor, I need to change some of my application's callback URLs to be the public `.githubpreview.dev` host instead of `localhost`.

Thanks!

**Comment by Tradersfamilyid** [comment]
From login

**Reply by Tradersfamilyid** [reply]
@bruce

**Comment by haltcase** [comment]
I'm looking for the same thing. I'm trying to find a way to avoid the manual change of things like callback URLs that must be different depending on desktop vs web VS Code.
